### PR TITLE
New package: pdf2djvu-0.9.19

### PR DESCRIPTION
**Pull request by spaskalev**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/pdf2djvu/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdf2djvu'
+pkgname=pdf2djvu
+version=0.9.19
+revision=1
+#archs="i686 x86_64"
```
Comment (classabbyamp) [inline srcpkgs/pdf2djvu/template]: remove all of these commented variables

File: srcpkgs/pdf2djvu/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#wrksrc=
+#create_wrksrc=yes
+build_style=gnu-configure
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="pkg-config"
+makedepends="djvulibre djvulibre-devel libdjvulibre poppler poppler-devel GraphicsMagick libgraphicsmagick libgraphicsmagick-devel exiv2 exiv2-devel libuuid libuuid-devel gettext gettext-devel"
```
Comment (paper42) [inline srcpkgs/pdf2djvu/template:7]: most of these are redundant, see https://github.com/void-linux/void-packages/pull/29996

File: srcpkgs/pdf2djvu/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+depends="djvulibre libdjvulibre poppler GraphicsMagick libgraphicsmagick exiv2 libuuid gettext"
```
Comment (paper42) [inline srcpkgs/pdf2djvu/template:8]: all of the required dependencies should be autodetected, remove this line

File: srcpkgs/pdf2djvu/patches/fix-c17.patch [hunk 4]
```diff
@@ -0,0 +1,11 @@
+--- a/autoconf.mk.in  2022-08-09 23:28:01.000000000 +0300
++++ b/autoconf.mk.in2 2022-08-12 01:16:49.770542275 +0300
+@@ -13,7 +13,7 @@
+ 
+ # Compiler, etc.:
+ CXX = @CXX@
+-CXXFLAGS = @OPENMP_CXXFLAGS@ @CXXFLAGS@ @DJVULIBRE_CFLAGS@ @POPPLER_CFLAGS@ @GRAPHICSMAGICK_CFLAGS@ @XMP_CFLAGS@
++CXXFLAGS = -std=c++17 @OPENMP_CXXFLAGS@ @CXXFLAGS@ @DJVULIBRE_CFLAGS@ @POPPLER_CFLAGS@ @GRAPHICSMAGICK_CFLAGS@ @XMP_CFLAGS@
```
Comment (classabbyamp) [inline srcpkgs/pdf2djvu/patches/fix-c17.patch:8]: this can be done by adding `export CXXFLAGS="-std=c++17"` in the template

**Review by classabbyamp — COMMENTED** [review]
```
hostmakedepends="pkg-config djvulibre gettext"
makedepends="djvulibre-devel poppler-devel libgraphicsmagick-devel exiv2-devel libuuid-devel"
```
this should satisfy all the dependencies

File: srcpkgs/pdf2djvu/patches/fix-c17.patch (hunk 4 above)
Comment (spaskalev) [reply srcpkgs/pdf2djvu/patches/fix-c17.patch:8]: Won't this override the rest of the CXXFLAGS ?

Comment (classabbyamp) [reply srcpkgs/pdf2djvu/patches/fix-c17.patch:8]: no

File: srcpkgs/pdf2djvu/template (hunk 2 above)
Comment (spaskalev) [reply srcpkgs/pdf2djvu/template]: Thanks, fixed

File: srcpkgs/pdf2djvu/template (hunk 3 above)
Comment (spaskalev) [reply srcpkgs/pdf2djvu/template]: Thanks, fixed

File: srcpkgs/pdf2djvu/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'pdf2djvu'
+pkgname=pdf2djvu
+version=0.9.19
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config djvulibre gettext"
+makedepends="djvulibre-devel poppler-devel libgraphicsmagick-devel exiv2-devel libuuid-devel"
+short_desc="Create DjVu files from PDF files"
+maintainer="Stanislav Paskalev <spaskalev@protonmail.com>"
+license="GPL-2.0-or-later"
```
Comment (classabbyamp) [inline srcpkgs/pdf2djvu/template:10]: `GPL-2.0-only`

```
 * pdf2djvu is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License version 2 as
 * published by the Free Software Foundation.
```

**Pull request merged**